packages/attestation-service/src/request.ts

```diff
@@ -82,7 +82,8 @@ export function respondWithAttestation(
   res: express.Response,
   attestation: AttestationModel,
   alwaysSuccess?: boolean | undefined,
-  salt?: string | undefined
+  salt?: string | undefined,
+  showAttestationCode?: boolean | undefined
```

Comment:
This feels a bit dangerous for me to include the attestation code with a simple boolean variable. I 'm worried that someone down the line will accidentally enable it in the public endpoint. It might be nice to merge the payload by passing it in explicitly?